__tests__/app.test.js

```diff
@@ -3,21 +3,34 @@ const request = require("supertest");
 const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const data = require("../db/data/test-data/index.js");
+const endpoints = require("../endpoints.json");
 
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
-describe('GET /api/topics', () => {
-    test("GET 200: - returns an array of objects containing slug and description properties", () => {
-        return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then((response) => {
-            expect(response.body.topics.length).toBe(3);
-            response.body.topics.forEach((topic) => {
-                expect(typeof topic.description).toBe("string");
-                expect(typeof topic.slug).toBe("string");
-            })
-        })
-    })
-})
+describe("GET: /api/topics", () => {
+  test("GET: 200 - returns an array of objects containing slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3);
+        response.body.topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+});
+
+describe("GET: /api", () => {
+  test("GET: 200 - returns with valid endpoints that are available for users", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(endpoints);
+        expect(typeof body.endpoints).toBe("object");
```

Comment:
A nice simple solution to this challenge. Often students overcomplicate this ticket